modules/decima-ext-model-exporter/src/main/java/com/shade/decima/ui/data/viewer/model/dmf/DMFModel.java

```diff
@@ -17,4 +17,8 @@ public void setSkeleton(@NotNull DMFSkeleton skeleton, @NotNull DMFSceneFile sce
             scene.skeletons.add(skeleton);
         skeletonId = scene.skeletons.indexOf(skeleton);
     }
+
+    public boolean isEmpty() {
+        return children.isEmpty() && (mesh==null || mesh.primitives.isEmpty());
+    }
```

Comment:
Overriden methods must be annotated with the `@Override` annotation and must call the parent method instead of copying its logic over all children.

```suggestion
    @Override
    public boolean isEmpty() {
        return super.isEmpty() && (mesh==null || mesh.primitives.isEmpty());
    }
```

Same for all other types that override this method